File: Java Programming/Competitive Programming/Coding Challenges 10/program1.java
//Q1.Write a program which accept one number from user and print that number of even numbers on screen.

//Input : 7

//Output: 2 4 6 8 10 12 14

import java.util.Scanner;

class program1
{
    static void PrintEven(int iNo)
    {
        int iCnt = 0;

        if (iNo <= 0)
        {
            return;
        }

        for (iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print(iCnt * 2 + "\t");
        }
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("Enter number: ");
        iValue = sobj.nextInt();

        PrintEven(iValue);

        sobj.close();
    }
}

File: C++ Programming/Competitive Programming/Coding Challenges 9/program1.cpp
//Q1.Write a program which accept number from user and print that number of $ & * on screen.
//Input : 5
//Output : $ * $ * $ * $ * $ *

//Input : 3
//Output : $ * $ * $ *

//Input : -3
//Output : $ * $ * $ *

#include <iostream>
using namespace std;

void Pattern(int iNo)
{
    if (iNo < 0)
    {
        iNo = -iNo;
    }

    while (iNo != 0)
    {
        cout << "$ *\t";
        iNo--;
    }
}

int main()
{
    int iValue = 0;

    cout << "Enter Number ";
    cin >> iValue;

    Pattern(iValue);

    return 0;
}